### PR TITLE
feat: release 2.13.0

### PR DESCRIPTION
**Pull request by spacewander**
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by tzssangglass — APPROVED** [review]

**Review by leslie-tsang — APPROVED** [review]

**Review by tokers — DISMISSED** [review]

**Comment by spacewander** [comment]
Let's cherry pick https://github.com/apache/apisix/pull/6663 before releasing.

**Review by juzhiyuan — APPROVED** [review]
LGTM

File: docs/zh/latest/discovery/kubernetes.md [hunk 1]
```diff
@@ -93,28 +98,89 @@ discovery:
...
+        - ip: "10.5.10.110"
+      ports:
+        - port: 3306
+  ```
 
-如果 kubernetes Endpoint 没有定义 portName, Kubernetes 服务发现插件会依次使用 targetPort, port 代替
+  nodes("default/plat-dev:3306") 调用会得到如下的返回值:
 
-# Q&A
+  ```
```
Comment (juzhiyuan) [inline docs/zh/latest/discovery/kubernetes.md:136]: Hi, do we need to add `lua` here?

Comment (leslie-tsang) [reply docs/zh/latest/discovery/kubernetes.md:136]: Maybe `json` would be better?

Comment (spacewander) [reply docs/zh/latest/discovery/kubernetes.md:136]: I prefer to solve it in the master branch? And would be good to check it in the CI if the mark is required.

Comment (leslie-tsang) [reply docs/zh/latest/discovery/kubernetes.md:136]: > I prefer to solve it in the master branch? And would be good to check it in the CI if the mark is required.

Agree.

Comment (juzhiyuan) [reply docs/zh/latest/discovery/kubernetes.md:136]: agree

**Pull request closed without merging**